### ISSUE TITLE
Can't build on Windows

### ISSUE THREAD

**Issue by SteveRussell33** [schnommus/verilog-vcvrack#1]
Cloned repo, cloned verilator repo into plugin root folder and installed, manually setting ``export VERILATOR_ROOT=`pwd` `` as per https://verilator.org/guide/latest/install.html
Trying to ``make`` plugin fails

```
mkdir -p build; \
verilator --cc rtl/core.sv -Mdir build/rtl-verilated -CFLAGS -fPIC
%Error: verilator/include/verilated_std.sv:28:9: Redeclaring the 'std' package is not allowed
   28 | package std;
      |         ^~~
        ./verilator/include/verilated_std.sv:2:1: ... note: In file included from verilated_std.sv
%Error: Exiting due to 1 error(s)
make: *** [Makefile:32: build/rtl-verilated/Vcore.mk] Error 1
```

**Comment by vk2seb** [comment]
Hey there, thanks for trying this out!

I don't have a Windows machine so it is a bit difficult for me to support here. But I just had a go in a Windows VM and it seems to compile under MSYS2 as documented in the VCV Rack guide with some tweaks.

I used verilator from the windows build of this oss-cad-suite release: https://github.com/YosysHQ/oss-cad-suite-build/releases/tag/2023-04-06

In the Makefile I used:
- VERILATOR_ROOT=/c/Users/<...>/oss-cad-suite/share/verilator
- I also had to add `$(VERILATOR_ROOT)/include/verilated_threads.cpp` to SOURCES, apparently windows verilator distro needs this.

In the shell environment I made sure the oss-cad-suite Verilator was in PATH
- `$ export PATH=$PATH:/c/Users/schno/Downloads/oss-cad-suite/bin`

With these changes I was able to build the .dll (did not try to run it as I don't have VCV installed on the windows machine)

![image](https://user-images.githubusercontent.com/1386830/230501557-fce52aec-355e-43ff-9d84-01cca6d245d5.png)

**Comment by SteveRussell33** [comment]
OK thanks for that, I will try again when I have time and will report back here how I get on.

**Comment by SteveRussell33** [comment]
Hi Seb,

Again thanks for the help, that worked out perfectly for me. 
I pointed ``VERILATOR_ROOT`` to the cloned repo of verilator and added the SOURCES line into the makefile as you suggested. The ``export`` binary path I added to my ``.bashrc`` file. This allowed the ``make dist`` plugin build to compile and create a ``dist`` folder with the folder/.vcvplugin files.

Later on I'll make a PR for your ``README`` file with a note on building on Windows for those that come across this project.

Cheers
Steve
